integration-tests/contract-tx-rollup-origination-operation.spec.ts

```diff
@@ -2,15 +2,15 @@ import { CONFIGS } from "./config";
 import { OpKind, Protocols } from "@taquito/taquito";
 
 CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
-  const jakartanet = (protocol === Protocols.PtJakart2) ? it : it.skip;
+  const limanet = (protocol === Protocols.PtLimaPtL) ? it : it.skip;
   const Tezos = lib;
   describe(`Test tx rollup origination using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup(true)
       done()
     })
-    jakartanet('should succeed to originate a rollup with auto-estimate of the fees', async (done) => {
+    limanet('should succeed to originate a rollup with auto-estimate of the fees', async (done) => {
```

Comment:
Should these tests run on all testnets? Using `it` rather than `limanet`?
I am not too sure why they were not running on Kathmandu..